### PR TITLE
stackrox: rox-22005 restore rox cluster pool

### PR DESCRIPTION
**Pull request by davdhacs**
Restoring the cluster pool for testing acs step-registry install. Keeping as-is on ocp4.13 because that version is still supported.

### CODE REVIEW

**Review by BradLugo — APPROVED** [review]
/lgtm

File: clusters/hive/pools/stackrox/ocp-4.13_clusterpool.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  creationTimestamp: null
+  labels:
+    architecture: amd64
+    cloud: aws
+    owner: stackrox
+    product: ocp
+    version: "4.13"
```
Comment (tommartensen) [inline clusters/hive/pools/stackrox/ocp-4.13_clusterpool.yaml:10]: [nit] you might choose a newer OCP version.

File: clusters/hive/pools/stackrox/ocp-4.13_clusterpool.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    version_lower: 4.13.0-0
+    version_upper: 4.14.0-0
+  name: stackrox-ocp-4-13-cluster-pool
+  namespace: stackrox-cluster-pools
+spec:
+  baseDomain: ocp-4-13.osci-pools.rox.systems
+  hibernationConfig: null
+  imageSetRef:
+    name: ocp-release-4.13.21-x86-64-for-4.13.0-0-to-4.14.0-0
+  installConfigSecretTemplateRef:
```
Comment (droslean) [inline clusters/hive/pools/stackrox/ocp-4.13_clusterpool.yaml:20]: Is this require a secret or there is another field to specify a configMap?

Comment (droslean) [reply clusters/hive/pools/stackrox/ocp-4.13_clusterpool.yaml:20]: Lets switch the object back to Secret and we can merge.

**Pull request merged**